packages/manager/core/generator/app/templates/src/components/layout-helpers/Dashboard/Dashboard.tsx

```diff
@@ -22,23 +20,25 @@ export type DashboardLayoutProps = {
 };
 
 const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs }) => {
-  const [panel, setActivePanel] = useState('');
+  const [activeTab, setActiveTab] = useState<string>('');
```

Comment:
shouldn't we initialize to the first tab ?

Reply:
We should'nt initialize cause we didn't want /Tabs1 in the url, by default it display index.tsx file